test/docker_audit.sh

```diff
@@ -71,11 +71,11 @@ ops_logs_before=$( get_logs_count $esopspod '/.operations.*/' )
 os::log::info "ops diff before:  $ops_logs_before"
 os::log::info "proj diff before: $logs_before"
 
-# ping,create,attach,start,delete generates 5 docker audit messages
+# ping,create,attach,start generates 4 docker audit messages
```

Comment:
What happened to `delete`?  did it just disappear?

Reply:
See the below comparison, both cases run this command `docker run -it --rm centos:7 echo "foo"`
docker-1.12.6-71.git3e8e77d.el7.x86_64:
====
create, attach, start, resize, wait, remove

docker-1.13.1-53.git774336d.el7.x86_64:
====
_ping, create, attach, start, resize